src/aircraft/Toliss_A330.ts

```diff
@@ -0,0 +1,81 @@
+import { AircraftConfigSimType, AircraftFeature, FeatureType } from '../defs'
+import {
+  AircraftConfig,
+  FeatureAddresses,
+  FeatureState,
+  FlapNames,
+  Meta,
+} from '../interface/aircraft'
+
+export default class Toliss_A330 extends AircraftConfig {
+  meta: Meta = {
+    id: 'toliss_a330',
+    name: 'ToLiss Airbus A330',
+    sim: AircraftConfigSimType.XPlane,
+    enabled: true,
+    priority: 2,
+    author: 'B.Fatih KOZ <https://github.com/FatihKoz>',
+  }
+
+  features: FeatureAddresses = {
+    [AircraftFeature.BeaconLights]: {
+      'AirbusFBW/OHPLightSwitches': FeatureType.IntArray,
+    },
+    [AircraftFeature.LandingLights]: {
+      'AirbusFBW/OHPLightSwitches': FeatureType.IntArray,
+    },
+    [AircraftFeature.NavigationLights]: {
+      'AirbusFBW/OHPLightSwitches': FeatureType.IntArray,
+    },
+    [AircraftFeature.StrobeLights]: {
+      'AirbusFBW/OHPLightSwitches': FeatureType.IntArray,
+    },
+    [AircraftFeature.TaxiLights]: {
+      'AirbusFBW/OHPLightSwitches': FeatureType.IntArray,
+    },
+    [AircraftFeature.APU]: {
+        'AirbusFBW/APUAvail': FeatureType.Int,
+    },
+  }
+
+  flapNames: FlapNames = {
+    0: 'UP',
+    1: 'CONF 1',
+    2: 'CONF 2',
+    3: 'CONF 3',
+    4: 'FULL',
+  }
+
+  match(title: string, icao: string, config_path: string): boolean {
+    return ['a330-941'].every((w) => title.includes(w))
```

Comment:
This will do:

```ts
return title.includes('a330-941')
```

Also, do `icao`/`config_path` return anything? Gotta check that on the PMDG, too

Reply:
So, is it wrong to have only one variable to check ? 

Should we provide an array there, like `['toliss', 'airbus', 'a330-941']`

Reply:
Aircraft description is like below in the acf file.
`P acf/_descrip A330-941`

Reply:
If it's only one thing, then doing it like above is better. I was doing it that other way when there were multiple terms

Reply:
> Aircraft description is like below in the acf file. `P acf/_descrip A330-941`

Is this being returned in either the config/icao?

Reply:
No, this is the **aircraft description** of XP12, which you read to match (as far as I understood from other files of XP11/12)